lib/config/index.js

```diff
@@ -18,9 +19,15 @@ const debugConfig = {
 // Get version string from package.json
 const {version} = require(path.join(appRootPath, 'package.json'))
 
+const commitID = getGitCommit(appRootPath)
+const sourceURL = getGitHubURL(commitID || version)
```

Comment:
`repo` paramter is not specified. In the current implementation, `sourceURL` will be `CommitID/tree/undefined`.

Reply:
That's correct, I already noticed that and currently think about the best way to get the real upstream repo for a branch out of the existing repo settings without re-build the whole git tooling on this